fv/test-connection/test-connection.go

```diff
@@ -308,6 +308,7 @@ func tryConnect(remoteIPAddr, remotePort, sourceIPAddr, sourcePort, protocol str
 	tc, err := NewTestConn(remoteIPAddr, remotePort, sourceIPAddr, sourcePort, protocol,
 		time.Duration(seconds)*time.Second, sendLen, recvLen)
 	if err != nil {
+		os.Stdout.WriteString(err.Error())
```

Comment:
Would prefer if we outputted a RESULT= line like we do on success in this case; have been trying to funnel all results through that mechanism.